### ISSUE TITLE
createArrowFiles：simpleError in H5Fopen(file, flags = flags, fapl = fapl, native = native): HDF5. File accessibility. Unable to open file

### ISSUE THREAD

**Issue by ZengTaox** [GreenleafLab/ArchR#1121]
Hello, when I use ArchR for scATAC-seq in windows, the errors comes. I had  set `subThreading = FALSE` and `threads = 1` in `createArrowFiles` but the error still comes.
```{R}
library(ArchR)
h5disableFileLocking()
addArchRGenome("hg19")
addArchRThreads(threads = 1)
inputFiles <- getTutorialData('Hematopoiesis')
rhdf5::h5errorHandling('verbose')
ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  filterTSS = 4, #Dont set this too high because you can always increase later
  filterFrags = 1000, 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE,
  subThreading = FALSE,
  threads = 1
)

************************************************************
2021-10-29 21:49:39 : ERROR Found in .tabixToTmp for (scATAC_BMMC_R1 : 1 of 3) 
LogFile = ArchRLogs\ArchR-createArrows-19c0164c5f4e-Date-2021-10-29_Time-21-49-39.log

<simpleError in H5Fopen(file, flags = flags, fapl = fapl, native = native): libhdf5
    error #000: C:/rtools40/tmp/CMake-hdf5-1.10.7/hdf5-1.10.7/src/H5F.c in H5Fopen(): line 429
        class: HDF5
        major: File accessibility
        minor: Unable to open file
    error #001: C:/rtools40/tmp/CMake-hdf5-1.10.7/hdf5-1.10.7/src/H5Fint.c in H5F_open(): line 1644
        class: HDF5
        major: File accessibility
        minor: Unable to open file
    error #002: C:/rtools40/tmp/CMake-hdf5-1.10.7/hdf5-1.10.7/src/H5FD.c in H5FD_open(): line 741
        class: HDF5
        major: Virtual File Layer
        minor: Unable to initialize object
    error #003: C:/rtools40/tmp/CMake-hdf5-1.10.7/hdf5-1.10.7/src/H5FDsec2.c in H5FD__sec2_open(): line 360
        class: HDF5
        major: File accessibility
        minor: Unable to open file>

************************************************************
```
**sessionInfo:**
```{R}
R version 4.1.1 (2021-08-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.936  LC_CTYPE=Chinese (Simplified)_China.936   
[3] LC_MONETARY=Chinese (Simplified)_China.936 LC_NUMERIC=C                              
[5] LC_TIME=Chinese (Simplified)_China.936    

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ArchR_1.0.1                       Rsamtools_2.10.0                 
 [3] BSgenome.Hsapiens.UCSC.hg19_1.4.3 BSgenome_1.62.0                  
 [5] rtracklayer_1.54.0                Biostrings_2.62.0                
 [7] XVector_0.34.0                    magrittr_2.0.1                   
 [9] rhdf5_2.36.0                      Matrix_1.3-4                     
[11] data.table_1.14.2                 SummarizedExperiment_1.24.0      
[13] Biobase_2.54.0                    GenomicRanges_1.46.0             
[15] GenomeInfoDb_1.30.0               IRanges_2.28.0                   
[17] S4Vectors_0.32.0                  BiocGenerics_0.40.0              
[19] MatrixGenerics_1.6.0              matrixStats_0.61.0               
[21] ggplot2_3.3.5                    

loaded via a namespace (and not attached):
 [1] pkgload_1.2.2            assertthat_0.2.1         BiocManager_1.30.16     
 [4] GenomeInfoDbData_1.2.7   yaml_2.2.1               remotes_2.4.0           
 [7] sessioninfo_1.1.1        pillar_1.6.4             lattice_0.20-44         
[10] glue_1.4.2               colorspace_2.0-2         XML_3.99-0.8            
[13] pkgconfig_2.0.3          devtools_2.4.2           zlibbioc_1.40.0         
[16] purrr_0.3.4              scales_1.1.1             processx_3.5.2          
[19] BiocParallel_1.28.0      tibble_3.1.5             generics_0.1.1          
[22] usethis_2.0.1            ellipsis_0.3.2           cachem_1.0.6            
[25] withr_2.4.2              cli_3.0.1                crayon_1.4.1            
[28] memoise_2.0.0            ps_1.6.0                 fs_1.5.0                
[31] fansi_0.5.0              pkgbuild_1.2.0           Cairo_1.5-12.2          
[34] tools_4.1.1              prettyunits_1.1.1        BiocIO_1.4.0            
[37] lifecycle_1.0.1          stringr_1.4.0            Rhdf5lib_1.14.2         
[40] munsell_0.5.0            DelayedArray_0.20.0      callr_3.7.0             
[43] compiler_4.1.1           rlang_0.4.11             grid_4.1.1              
[46] RCurl_1.98-1.5           rhdf5filters_1.4.0       rstudioapi_0.13         
[49] rjson_0.2.20             bitops_1.0-7             testthat_3.0.4          
[52] restfulr_0.0.13          gtable_0.3.0             DBI_1.1.1               
[55] R6_2.5.1                 GenomicAlignments_1.30.0 dplyr_1.0.7             
[58] fastmap_1.1.0            utf8_1.2.2               rprojroot_2.0.2         
[61] desc_1.3.0               stringi_1.7.5            parallel_4.1.1          
[64] Rcpp_1.0.7               vctrs_0.3.8              tidyselect_1.1.1    
```
